modules/nlu/src/backend/module-lifecycle/on-bot-mount.ts

```diff
@@ -39,8 +37,9 @@ export function getOnBotMount(state: NLUState) {
           return
         }
 
-        const intentDefs = await getIntents(ghost)
-        const entityDefs = await entityService.getCustomEntities()
+        const api = await createApi(bp, botId)
+        const intentDefs = await api.fetchIntents()
```

Comment:
Is there any chance this operation fails? Maybe there should be a catch or something to at least log an error about an error while fetching intents and entities?

I don't have a strong opinion about this, just asking... what do you think?

Reply:
I'd say it's not really necessary. We're calling our own server so something going wrong is unlikely